simpledotorg/simple-server
# 2116 password validation checklist

**Story card:** [ch2116](https://app.clubhouse.io/simpledotorg/story/2116/update-ui)

## Because

We want to give users live feedback on their password validity. 

## This addresses

This is the final piece in changing our password scheme. 

## Test instructions

1) invitations
- invite a new user through the admin dashboard
- get the invitation link from rails logs or mailcatcher
- go to the page and confirm that the checklist accurately reflects the current state of the password
- confirm that the form button becomes submittable when the password is valid and that submission works

2) changing password
- log out of simple and click "forgot password". enter your user's email address
- get the invitation link from rails logs or mailcatcher
- go to the page and confirm that the checklist accurately reflects the current state of the password
- confirm that the form button becomes submittable when the password is valid and that submission works
